src/App.js

```diff
@@ -8,16 +8,19 @@ import BottomNavBar from "./common/BottomNavBar";
 import Home from "./home";
 import Navigation from './navigation';
 import Account from "./account";
+import { useAuthentication } from "./common/useAuthentication";
 
 function App() {
+  const {user} = useAuthentication();
+
   return (
     <Router>
       <div className="flex flex-col min-h-screen">
         <div className="flex-grow">
           <Routes>
             <Route path="/" element={<Home />} />
             <Route path="/navigation" element={<Navigation />} />
-            <Route path="/account" element={<Account />} />
+            <Route path="/account" element={<Account/>}/>
```

Comment:
Please pay attention to the code specification. Why did you delete the spaces?

Reply:
Whoops. Will revert to the original.